scala/scala_toolchain.bzl

```diff
@@ -1,8 +1,13 @@
 def _scala_toolchain_impl(ctx):
-  toolchain = platform_common.ToolchainInfo(scalacopts = ctx.attr.scalacopts,)
+  toolchain = platform_common.ToolchainInfo(
+      scalacopts = ctx.attr.scalacopts,
+      unused_dependency_checker_mode = ctx.attr.unused_dependency_checker_mode)
   return [toolchain]
 
 scala_toolchain = rule(
-    _scala_toolchain_impl, attrs = {
+    _scala_toolchain_impl,
+    attrs = {
         'scalacopts': attr.string_list(),
+        'unused_dependency_checker_mode': attr.string(
+            default = "off", values = ["off", "warn", "error"]),
```

Comment:
since this has a default of "off", won't this make the setting on the scala_provider unused?

Reply:
The setting in the params passed to individual targets (not in the scala_provider) are checked first and if they're not set then the toolchain is used